demos/default/next.config.js

```diff
@@ -89,5 +89,6 @@ module.exports = {
   },
   experimental: {
     optimizeCss: false,
+    incrementalCacheHandlerPath: require.resolve('./cache-handler.js'),
```

Comment:
This won't be hard-coded. This is hard-coded for the moment for testing purposes. The real implementation will need to generate this and modify the Next.js config, pre-build.